engine/runtime-compiler/src/main/scala/org/enso/compiler/context/InlineContext.scala

```diff
@@ -25,9 +25,15 @@ case class InlineContext(
   freshNameSupply: Option[FreshNameSupply]     = None,
   passConfiguration: Option[PassConfiguration] = None,
   pkgRepo: Option[PackageRepository]           = None
-) {
+) extends AutoCloseable {
```

Comment:
Isn't it easier to make the `InlineContext` closable than wrap it into `InlineContextResource`?